apidoc/Titanium/UI/iOS/Stepper.yml

```diff
@@ -0,0 +1,180 @@
+---
+name: Titanium.UI.iOS.Stepper
+summary: |
+    A widget used to increment and decrement a value
+description: |
+    <table id="IOS Stepper">
+      <tr>
+        <td><img src="images/stepper/stepper.png" height="50" /></td>
+        <td><img src="images/stepper/stepper_customImages" height="51" /></td>
+      </tr>
+      <tr><th colspan="2">iOS</th></tr>
+    </table>
+
+    A Stepper displays its selected state while it is being pressed.
+    You can specify background images for each state on iOS.
+    Use the <Titanium.UI.createstepper> method or a **`<stepper>`** Alloy element to create a stepper 
+    (see [Examples](#!/api/Titanium.UI.iOS.stepper-examples)) .
+
+    * When assigning a custom image to the `backgroundImage` property, the image must be
+    partially or wholly transparent for the background color or background gradient to be visible.
+    * Similarly, for an assigned background gradient to show through, the `backgroundColor` may need to be
+    set to 'transparent'.
+
+    *Note that the stepper cannot be re-sized.
+
+extends: Titanium.UI.View
+since: "5.4.0"
+excludes:
+    events: [dblclick, doubletap, focus, keypressed, longclick, longpress, pinch, postlayout,
+             singletap, swipe, touchcancel, touchend, touchmove, touchstart, twofingertap]
+properties:
+  - name: value
+    summary: The current value of the stepper.
+    type: Number
+
+  - name: continous
+    summary: |
+        If YES, value change events are sent immediately when the value changes during user interaction.
+        If NO, a value change event is sent when user interaction ends.
+        The default value is YES.
+    type: Boolean
+
+  - name: autorepeat
+    summary: |
+        If YES, the user pressing and holding on the stepper repeatedly alters value.
+        The default value is YES.
+    type: Boolean
+
+  - name: wraps
+    summary: |
+        If YES, incrementing beyond maximumValue sets value to minimumValue; likewise, decrementing below
```

Comment:
Code-highlight `maximumValue ` and `minimumValue `. Use A point instead of a semicolon.